DOCUMENTATION.md

```diff
@@ -52,7 +52,7 @@ Name | Type | Description | `flatpak override` equivalent
 GPU acceleration | Toggle | Allow the application to access the graphics direct rendering to take advantage of GPU acceleration. | `--device=dri` and `--nodevice=dri`
 Virtualization | Toggle | Allow the application to support virtualization. | `--device=kvm` and `--nodevice=kvm`
 Shared memory | Toggle | Allow the application to access shared memory. | `--device=shm` and `--nodevice=shm`
-All devices | Toggle | Allow the application to access to all devices, such as webcam and external devices. | `--device=all` and `--nodevice=all`
+All devices | Toggle | Allow the application to access all devices, such as webcam, microphone and external devices. <br /> <br /> For example, if it's disabled for Element, it will no longer be possible to do voice or video calls with this application. | `--device=all` and `--nodevice=all`
```

Comment:
A small comment here, for historical reasons, microphone access is handled by the PulseAudio permission.

Read [this](https://github.com/jdstrand/snapd/commit/ad00b44cffd2e1f18fe4d45ac11d3fd84a2b7417) if you're curious for the details.

So this example shouldn't mention the microphone  :)

Reply:
ahh i see! wasn't aware of that, thanks. have fixed it now.